tests/cypress/support/commands.ts

```diff
@@ -97,6 +98,9 @@ Cypress.Commands.add('addFleetGitRepo', ({ repoName, repoUrl, branch, path, gitO
   if (path) {
     cy.addPathOnGitRepoCreate(path);
   }
+  if (path2) {
+    cy.addPathOnGitRepoCreate(path2, 1);
```

Comment:
do we need `path2`?
You added an `index` flag that by default points to 0 unless specified otherwise.

Reply:
Yes, we require that `index` to point to each `array-list-box` input field. See below screenshot for more details.

<details>
<summary>Screenshot showing locator for multiple paths (more than 1 path.)</summary>

![Screenshot from 2024-07-30 21-36-28](https://github.com/user-attachments/assets/0de92033-a04c-4bd6-b94e-611b536e46e1)

</details>